[Docs]-update-process-arguments-scrubbing-helm (#10598)
DataDog/documentation: content/en/infrastructure/process.md

    strip_proc_arguments: true
```

{{% /tab %}}

{{% tab "Helm" %}}

> You can use the Helm chart to define your own list to be merged with the default one by updating your `datadog-values.yaml` by adding the env variables `DD_SCRUB_ARGS` and `DD_CUSTOM_SENSITIVE_WORDS`, then upgrade your Datadog Helm chart:

Reviewer: ```suggestion
You can use the Helm chart to define your own list, which will be merged with the default one. Add the environment variables `DD_SCRUB_ARGS` and `DD_CUSTOM_SENSITIVE_WORDS` to your `datadog-values.yaml` file, and upgrade your Datadog Helm chart:
```

```yaml
datadog:
    # (...)
    processAgent:
        enabled: true
        processCollection: true
    agents:
        containers:
            processAgent:
                env:
                - name: DD_SCRUB_ARGS 
                  value: "true"
                - name: DD_CUSTOM_SENSITIVE_WORDS
                  value: "personal_key,*token,*token,sql*,*pass*d*" 
```

Use wildcards (`*`) to define your own matching scope. However, a single wildcard (`'*'`) is not supported as a sensitive word.

Set `DD_SCRUB_ARGS` to `false` to completely disable the process arguments scrubbing.

> You can also scrub **all** arguments from processes by enabling the `DD_STRIP_PROCESS_ARGS` variable in your `datadog-values.yaml` configuration file:

Reviewer: ```suggestion
Alternatively, you can scrub **all** arguments from processes by enabling the `DD_STRIP_PROCESS_ARGS` variable in your `datadog-values.yaml` file:
```
